data/default-config.json

```diff
@@ -1,10 +1,31 @@
 {
-  "channel": "legi-bot",
+  "channel": "legisacn",
   "cron": "*/30 * * * *",
   "embeds": false,
   "interval": 60,
-  "prefix": "!",
+  "prefix": ">",
   "query": "\"right to repair\" OR \"right-to-repair\" OR ((servicing OR repair) AND electronics) OR (fair AND electronic AND repair OR independent)",
   "state": "ALL",
-  "debug": false
+  "debug": false,
+  "command-list": {
```

Comment:
setting name should match config command ("permissions")

Reply:
Also, default-config/permissions should fit into database/config/permissions, not database/permissions

Reply:
I did not put the permissions on the `default-config.json` because there was no need to, since the absence of configuration means that it is allowed by default, however I saw a need to have a list of available commands in order to validate the config. Without it the BOT would be vulnerable to a storage abuse by setting any commands that don't exist.

It also makes it clear that the command was set properly in case the person doing the command mistypes the command name.

Reply:
I'll take this "storage abuse" issue into account later :)